apps/knowledge-base/export-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "spaceId": "2fyan6uo0t86",
```

Comment:
I can't see this as a problem, to be honest... 

The process of importing/exporting is still manual and we want to keep it that way because it's much simpler and any change on the space template could mess up the installation process.

People using this app will have access to the same content anyway and they can make changes to this space only if they have a management token, which is not the case.